src/Scale.purs

```diff
@@ -201,16 +202,29 @@ instance scaleOrdinal :: Scale OrdinalScale where
   copy = unsafeCopy
   toFunction = unsafeToFunction
 
+unsafeInvert :: forall t. t
```

Comment:
This and the others could *probably* be constrained to Number right? I went the 'safe' route with the compiler suggestion.